README.md

```diff
@@ -1,40 +1,148 @@
+<!-- nextcloud-talk-jitzsi-bot README.md -->
+<!-- version: 0.1.1 -->
+
 # Nextcloud Talk Jitsi Bot
 
-A chat bot for Nextcloud Talk that creates [Jitsi](https://jitsi.org) video chat meetings. It also includes a framework for building Nextcloud Talk chatbots.
+A chat bot for Nextcloud Talk that creates [Jitsi](https://jitsi.org) video chat meetings.
+It also includes a framework for building Nextcloud Talk chatbots.
 
 ## Overview
 
-Nextcloud Talk Jitsi Bot creates ad-hoc Jitsi meetings from Nextcloud Talk. Just add the bot to the conversation, type `#videocall` or `#videochat` and get a link to a Jitsi meeting!
+Nextcloud Talk Jitsi Bot creates ad-hoc Jitsi meetings from Nextcloud Talk sessions.
+
+The talk sessions needs to be aware of the bot. Just create a new `Jitsi-Bot user` inside your trusted `nextcloud` domain.
```

Comment:
While I do appreciate the instructions, what does `domain` refer to in this specific instruction? Sound like some LDAP/AD stuff to me here

Reply:
> While I do appreciate the instructions, what does `domain` refer to in this specific instruction? Sound like some LDAP/AD stuff to me here

no, this is in line with nextcloud notation to address suitable domains, as defined in config.php.